.evergreen/run-tests.sh

```diff
@@ -22,9 +22,4 @@ OLD_PATH=$PATH
 PATH=/opt/php/${PHP_VERSION}-64bit/bin:$OLD_PATH
 
 # Run the tests, and store the results in a Evergreen compatible JSON results file
-case "$OS" in
-   *)
-      php vendor/bin/phpunit -v
-      ;;
-esac
-
+php vendor/bin/phpunit
```

Comment:
I assume removing verbose output was intentional. So long as we still get detail for skipped tests and failures, that works for me.

Reply:
We get details for failures, but not for skipped tests. I can add this back if you'd like.

Reply:
Realistically, I don't think we're ever going to look at build logs and investigate whether tests are skipped when they shouldn't be, so I'm fine leaving this as-is.

AFAIK PHPLIB does not suffer from the same issue that PHPC did to necessitate [PHPC-1343](https://jira.mongodb.org/browse/PHPC-1343) (a canary test that fails when no servers are available).